docs/docs/15-concepts.md

```diff
@@ -4,6 +4,12 @@ description: Find out more about key Kargo concepts - stages, freight, warehouse
 ---
 # Key Kargo Concepts
 
+:::note
+Documentation is Under Active Development
+
```

Comment:
This sounds (mildly) scary compared to the sentence below. I think it's a better tone overall if we leave this part out.

```suggestion
```